src/esbmc/esbmc_parseoptions.cpp

```diff
@@ -1684,6 +1685,22 @@ bool esbmc_parseoptionst::process_goto_program(
       return true;
     }
 
+    if(cmdline.isset("output-goto"))
+    {
+      msg.status("Writing GOTO program to file");
+      std::string fullname = cmdline.args[0];
```

Comment:
Would it be possible instead of interpreting `args[0]` to give `--output-goto` a parameter that specifies the output path?

Reply:
Oh, what a silly mistake I have made.  Thank you very much!